Polygon_mesh_processing/doc/Polygon_mesh_processing/PackageDescription.txt

```diff
@@ -150,6 +153,12 @@ and provides a list of the parameters that are used in this package.
 - `CGAL::Polygon_mesh_processing::max_distance_to_triangle_mesh()`
 - `CGAL::Polygon_mesh_processing::sample_triangle_mesh()`
 
+##Detect Features Functions ##
+- `CGAL::Polygon_mesh_processing::detect_features()`
+- `CGAL::Polygon_mesh_processing::detect_sharp_edges()`
+- `CGAL::Polygon_mesh_processing::detect_surface_patches()`
+- `CGAL::Polygon_mesh_processing::detect_vertices_incident_patches()`
```

Comment:
If there is no overload ambiguity, I agree with Jane. The fact that the incident is to the vertex is already encoded by the type.